pulseaudio_dbus.lua

```diff
@@ -180,6 +180,31 @@ function pulse.get_core(connection)
 end
 
 --- Pulseaudio
+-- [Stream](https://www.freedesktop.org/wiki/Software/PulseAudio/Documentation/Developer/Clients/DBus/Stream/)
+-- Use @{pulse.get_stream} to obtain a stream object.
+-- @type Stream
+pulse.Stream = {}
+
+--- Get the pulseaudio [stream object](https://www.freedesktop.org/wiki/Software/PulseAudio/Documentation/Developer/Clients/DBus/Stream/)
```

Comment:
Please use `[Stream]` rather than `[stream object]` (for consistency with the rest of the docs).